cmd/create/cluster/cmd.go

```diff
@@ -3033,6 +3094,41 @@ func run(cmd *cobra.Command, _ []string) {
 	}
 }
 
+func GetBillingAccountContracts(cloudAccounts []*accountsv1.CloudAccount,
+	billingAccount string) ([]*accountsv1.Contract, bool) {
+	var contracts []*accountsv1.Contract
+	var isContractEnabled bool
+	for _, account := range cloudAccounts {
+		if account.CloudAccountID() == billingAccount {
+			contracts = account.Contracts()
+			if ocm.HasValidContracts(account) {
+				isContractEnabled = true
+			}
+		}
+	}
+	return contracts, isContractEnabled
+}
+
+func GenerateContractDisplay(contract *accountsv1.Contract) string {
+	format := "Jan 02, 2006"
+	dimensions := contract.Dimensions()
+
+	numberOfVPCs, numberOfClusters := ocm.GetNumsOfVPCsAndClusters(dimensions)
```

Comment:
I believe you meant numberOfVCPUs, not VPCs?

Reply:
yep yep, will change the name